releasenotes/notes/40032.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: release-notes/v2
+kind: feature
+area: telemetry
+
+releaseNotes:
+  - |
+    **Added** support for use of the OpenTelemetry provider with the Telemetry API.
```

Comment:
should we point out it's OpenTelemetry trace provider?

Reply:
done.